modules/flyte/outputs.tf

```diff
@@ -0,0 +1,19 @@
+output "aks" {
```

Comment:
I believe the naming of this output is not reflective of what it is, the information is regarding `flyte` not aks. I believe it was mentioned on the eks PR.

Reply:
The outputs are now flattened, in the same manner as variables.  README.md now has an entry for each output defined.